cognee/infrastructure/llm/structured_output_framework/baml/baml_client/sync_client.py

```
@@ -89,106 +92,30 @@ def parse(self):
+        else:
+            # Original non-streaming code
+            result = self.__options.merge_options(baml_options).call_function_sync(
+                function_name="AcreateStructuredOutput",
+                args={
+                    "text_input": text_input,
+                    "system_prompt": system_prompt,
+                },
+            )
+            return typing.cast(
+                types.ResponseModel, result.cast_to(types, types, stream_types, False, __runtime__)
+            )
```

Pylint (3.3.8)

[refactor] Unnecessary "else" after "return", remove the "else" and de-indent the code inside it

(R1705)